.github/workflows/ci.yml

```diff
@@ -18,5 +18,7 @@ jobs:
         run: npm run lint:eslint
```

Comment:
I'll hold off merging this until [this gets in](https://github.com/BrightspaceUI/testing/pull/126) and I can update this script accordingly.